test/unit/app/controllers/permissions/permissions-controller-test.js

```diff
@@ -102,8 +102,10 @@ describe('permissions controller', function () {
       assert.deepEqual(cAccounts, [], 'origin should have no accounts')
     })
 
-    it('does not handle "MetaMask" origin as special case', async function () {
-      const metamaskAccounts = await permController.getAccounts('MetaMask')
+    it('does not handle "MetaMask" or "metamask" origins as special cases', async function () {
```

Comment:
Nit: the origin is always `metamask`. This test description was outdated - it used to be `MetaMask`.

```suggestion
    it('does not handle ""metamask" origin as special case', async function () {
```

Reply:
Oh, I see this was explicitly tested as well. Where does `MetaMask` come from? 😕

Reply:
I forget where I got that into my head, but it's fixed now!